### DISCUSSION TITLE
Youtube Video not Showing

### DISCUSSION THREAD

**Discussion by Seth-AIG** [requarks/wiki#3099] [Help / Questions]
Im using the insert media button at the top to add the URL of a youtube video. After I save the page, then reload it, the video does not show?

**Comment by johncpv1985** [comment]
> Im using the insert media button at the top to add the URL of a youtube video. After I save the page, then reload it, the video does not show?

I have the same problem. Neither this button nor the code works.

**Reply by lordmarqui** [reply]
Same issue

Any resolution?

I've followed all the suggestions to enable iframe emebedding. Nothing works.

**Reply by grahamgraham** [reply]
I'm using the latest, 2.5.305, and I am also having this issue.  I've read through [this issue](https://github.com/requarks/wiki/pull/2160), and I already have the iframe options they mentioned enabled. 

To be more clear, this only happens when using the Visual Editor and trying to embed to insert media. Raw HTML and Markdown work fine, but are less user friendly. 

@NGPixel Do you know if there's a resolution to this?

**Reply by bigman73** [reply]
Markdown doesn't work for me either, can you provide an example that works for you?
